twoliter/src/project.rs

```diff
@@ -102,10 +105,26 @@ impl Project {
         self.project_dir.clone()
     }
 
+    pub(crate) fn schema_version(&self) -> SchemaVersion<1> {
+        self.schema_version
+    }
+
     pub(crate) fn release_version(&self) -> &str {
         self.release_version.as_str()
     }
 
+    pub(crate) fn vendor(&self) -> BTreeMap<String, Vendor> {
+        self.vendor.clone()
```

Comment:
Seems a bit unidiomatic to do the clone here. Usually you return a `&` and let the caller do what they want.